### PR TITLE
update meta title and desc

### PR DESCRIPTION
**Pull request by jonnysmillie**

### CODE REVIEW

File: config/siteMetadata.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ const { WEBSITE_URL, DOCS_URL, MARKETPLACE_URL, CAREERS_URL } = require('./webCo
 module.exports = {
   menus,
   title: `Substrate_ Docs`,
-  title_meta: `Substrate enables developers to quickly and easily build future-proof blockchains optimized for any use case.`,
-  description: `Substrate enables developers to quickly and easily build future-proof blockchains optimized for any use case.`,
+  title_meta: `Quick Start Guide for Substrate - Build Today.`,
+  description: `All of the Substrate tutorials and how-to guides require you to build and run a Substrate node in your development environment.`,
```
Comment (rmnprkrl) [inline config/siteMetadata.js:8]: @jonnysmillie FYI
I'm checking on the text with marcomms, seems not well describing the project.

File: src/components/site/NavLegal.js [hunk 2]
```diff
@@ -17,24 +17,32 @@ const HeaderNav = () => {
         <Link
           className="mr-11 md:mr-0 md:ml-9 transform transition-all duration-300 ease-in-out hover:opacity-50"
           to={siteMetadata.element}
+          name="Element"
```
Comment (rcos7) [inline src/components/site/NavLegal.js:20]: Looks like name is deprecated, can be replaced with an id if we need it
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

File: src/components/site/SEO.js [hunk 3]
```diff
@@ -3,22 +3,21 @@ import { Helmet } from 'react-helmet';
...
-export default function SEO({ children = null, description = '', lang = 'en', meta = [], title }) {
+export default function SEO({ children = null, description, lang = 'en', meta = [], title, excerpt }) {
   const { siteMetadata } = useSiteMetadata();
 
-  const metaDescription = description || siteMetadata.description || '';
-  const defaultTitle = siteMetadata.title;
-  const titleMeta = siteMetadata.title_meta;
+  const metaDescription = description || excerpt || siteMetadata.description;
+  const metaTitle = siteMetadata.title_meta;
+  const defaultTitle = title + ' | ' + siteMetadata.title || siteMetadata.title_meta;
```
Comment (rcos7) [inline src/components/site/SEO.js:11]: nit: prefer template literals over concatenating strings

**Review by rcos7 — CHANGES_REQUESTED** [review]
Looks good, just a couple of comments

**Comment by rmnprkrl** [comment]
@jonnysmillie LGTM

**Pull request merged**